docs/UserGuide.md

```diff
@@ -275,6 +334,73 @@ Expected output:
 List sorted in alphabetical order of names
 ```
 
+### Viewing Statistics : `stats`
+Displays the requested statistic about the currently displayed list.
+
+Format: `stats FIELD`
+
+Parameters:`FIELD` 
+* The requested field whose statistic is to be calculated.
+* Valid fields that can be requested for are:
+  * `avail` :  shows the statistics of fosterers who are available to foster, and the animals they can foster.
+  * `current` : shows the statistics of fosterers who are currently fostering, and the animals they are fostering.
+  * `housing` : shows the statistics of the various housing types.
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Warning:Statistics are calculated based on the currently displayed list.**
```

Comment:
wait i thought we are changing warning to caution hahaha

Reply:
SORRY i just read the docs, changed it!